fastlane/Matchfile

```diff
@@ -13,6 +13,10 @@ for_lane :sync_signing_adhoc do
     force_for_new_devices true
 end
 
+for_lane :sync_signing do
+    readonly: is_ci
```

Comment:
What is this for? So that ad-hoc lane is read-write but when sync_signing is called alone (appstore release?) then it’s read-only?

Reply:
that is correct, do you want me to add a comment, or refactor it somehow?

Reply:
Actually a comment would be great, to point out that it’s read-write for the adhoc lane. Thanks!